app/src/main/java/org/simple/clinic/patient/businessid/BusinessId.kt

```diff
@@ -111,7 +111,7 @@ data class BusinessId(
       WHERE patientUuid = :patientUuid AND identifierType = :identifierType AND deletedAt IS NULL
       ORDER BY createdAt DESC LIMIT 1
       """)
-    fun latestIdForType(patientUuid: UUID, identifierType: IdentifierType): BusinessId?
+    fun latestIdForType(patientUuid: UUID, identifierType: IdentifierType): Flowable<List<BusinessId>>
```

Comment:
Rename this method to `latestForPatientOfType`?

Reply:
This name seems like there is a type of patient.
Let me think of something else.

Reply:
Maybe just `latestForPatient`?